liblouis/compileTranslationTable.c

```diff
@@ -546,26 +551,27 @@ getDots(widechar d, TranslationTableHeader *table) {
 	while (offset) {
 		TranslationTableCharacter *character =
 				(TranslationTableCharacter *)&table->ruleArea[offset];
-		if (character->realchar == d) return character;
+		if (character->value == d) return character;
 		offset = character->next;
 	}
 	return NULL;
 }
 
 static TranslationTableCharacter *
-putChar(const FileInfo *file, widechar c, TranslationTableHeader **table) {
+putChar(const FileInfo *file, widechar c, TranslationTableHeader **table,
+		TranslationTableOffset *characterOffset) {
```

Comment:
Strictly speaking afaict this change has nothing to do with the base opcode? It's just a bug fix, correct?

Reply:
I need the offset in order to link to the `basechar` from a `TranslationTableCharacter`.